.golangci.yaml

```diff
@@ -67,6 +67,9 @@ linters-settings:
 
 issues:
   exclude-rules:
+    - path: pkg/reporter
```

Comment:
These changes made the reporters big enough to trigger `dupl` - I've just disabled it for the whole package because while it's technically not wrong, the only way I know that we could address that would be by having a `standardReporter` that gets used by each struct with the basic "print to stdout and error" behaviour (like what I've done in `osv-detector` with the ["memory database"](https://github.com/G-Rath/osv-detector/blob/main/pkg/database/mem-check.go)), but from what I understand that would be a breaking change (because when initializing the struct you have to pass in the inner struct) and it won't actually result in much savings